browser/scripts/ui.js

```diff
@@ -502,30 +504,70 @@ VizorUI.prototype.setModeProgram = function() {
 	return true;
 };
 
-VizorUI.prototype.buildBreadcrumb = function(graph, beforeRender) {
+VizorUI.prototype.buildBreadcrumb = function(graph, selected, beforeRender) {
```

Comment:
since this doesn't actually use any UI bits, would be easy and very useful to write a unit test for this function.